ancar/portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,19 +30,23 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  ArrayList<String> commentsList = new ArrayList<>();
-
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String json = convertToJsonUsingGson(commentsList);  
-    response.setContentType("application/json;");
-    response.getWriter().println(json);
+    // String json = convertToJsonUsingGson(commentsList);  
+    // response.setContentType("application/json;");
+    // response.getWriter().println(json);
```

Comment:
do not submit commented out code... either you need to change this to work with the new stuff, or remove it if it is not needed... in case the effort would be significant, adding a TODO in the format of TODO(issue id) or TODO(username) if there is no issue for it, and the resolver will 100% not change can justify it for a short time